### PR TITLE
Add linting for ansible

### LINKED ISSUE #142: Use ansible lint in the CI for consistency across yaml files
### Context

Use https://github.com/ansible/ansible-lint in the CI

### Value and/or benefit

- Consistency in the yaml files

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by aktech**
## Reference Issues or PRs

Fixes #142 

This applied `ansible-lint --fix` for overall consistency in ansible files. I have skipped a bunch of linting tests in `.ansible-lint` so that we can fix them over time without introducing more linting problems.

Also skips the vagrant tests, until we fix it as it was never working anyways and gives a false view of failure.

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [x] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: playbook.yaml [hunk 1]
```diff
@@ -1,56 +1,58 @@
...
+    # plugins
+    - bodo
 
-   tasks:
-     - name: Additional ansible tasks
-       include_tasks: "{{ item }}"
-       with_items: "{{ additional_tasks | default([]) }}"
+  tasks:
+    - name: Additional ansible tasks
+      ansible.builtin.include_tasks: "{{ item }}"
```
Comment (viniciusdc) [inline playbook.yaml:48]: @aktech are those `ansible.builtin` comming from the linter as well?

Comment (aktech) [reply playbook.yaml:48]: Yep

**Comment by aktech** [comment]
@viniciusdc are you happy for me to merge this?

**Comment by viniciusdc** [comment]
Hi @aktech i was hoping to have a chance to test deploying those changes to double check if the variable renaming would interfere in anything

**Comment by viniciusdc** [comment]
But I am okay with merging and testing later, I can open another PR with fixes if needed

**Comment by aktech** [comment]
I just deployed it and it worked after a couple of issues, which I have fixed and pushed now to the PR.

**Review by viniciusdc — APPROVED** [review]
LGTM, thanks for checking the deployment @aktech

**Comment by aktech** [comment]
Thanks for the review @viniciusdc 🎉

**Pull request merged**